ubuntu22.04/precompiled/local-repo.sh

```diff
@@ -71,12 +76,28 @@ fetch_nvidia_installer () {
   rm $DRIVER_RUN_FILE.run
 }
 
+imex_install() {
+  if [ "$DRIVER_BRANCH" -ge "580" ]; then
+    apt-get install -y --no-install-recommends nvidia-imex=${DRIVER_VERSION}-1
+  elif [ "$DRIVER_BRANCH" -ge "570" ]; then
+    apt-get install -y --no-install-recommends nvidia-imex-${DRIVER_BRANCH}=${DRIVER_VERSION}-1;
+  fi
+}
+
+extra_pkgs_install() {
+      apt-get update
+      imex_install
+      rm -rf /var/lib/apt/lists/*
+}
+
 if [ "$1" = "download_driver_package_deps" ]; then
   download_driver_package_deps
 elif [ "$1" = "build_local_apt_repo" ]; then
   build_local_apt_repo
 elif [ "$1" = "fetch_nvidia_installer" ]; then
   fetch_nvidia_installer
+elif [ "$1" = "extra_pkgs_install" ]; then
```

Comment:
I don't think we need to add this method. All the package installation should be in the `download_driver_package_deps` method